### ISSUE TITLE
InfiniteChestsV3.dll Chest Limitation

### ISSUE THREAD

**Issue by Diospectre** [RenderBr/tShock-v5-plugins#10]
InfiniteChestsV3.dll does auto convert when the plugin is installed for the first time, thus chests modded by InfiniteChestsV3.dll can't be reconverted back to vanilla chests - if ones change their mind - if the world they're playing has more than 1.000 chests already. isn't it supposed to be 8.000 since Terraria v1.4?

**Comment by Maxthegreat99** [comment]
you are right, at this point its really just a limitation to the plugin itself, i can publish a quick fix for it, relatively quickly. Ill get to that wait.
